packages/modular-scripts/react-dev-utils/WebpackDevServerUtils.js

```diff
@@ -177,7 +179,7 @@ function createCompiler({
             'Files successfully emitted, waiting for typecheck results...',
           ),
         );
-      }, 100);
+      }, 0);
```

Comment:
Why do we have a timeout?

Reply:
In most cases the diagnostics should come back pretty quickly - so this timeout is in place to ensure we give TypeScript sufficient time to process the files.